### ISSUE TITLE
`sample_prior_predictive` for CLV models

### ISSUE THREAD

**Issue by wd60622** [pymc-labs/pymc-marketing#1155]
No description provided.

**Comment by ColtAllen** [comment]
I'm actually working on a PR right now that involves plotting prior predictives. This can't be done with the `ModelBuilder` methods due to the X-Y convention.

**Comment by wd60622** [comment]
Yup, understand that limitation. I plan to rework that in the near future
